### DISCUSSION TITLE
Dịch Normal Equation

### DISCUSSION THREAD

**Discussion by minhduc0711** [mlbvn/handson-ml2-vn#23] [Terminology] [answered]
Từ này mình thấy một vài nguồn không chính thống trên mạng dịch là `công thức chuẩn`.
Nhưng mà từ [góc nhìn đại số tuyến tính](https://stats.stackexchange.com/a/305748), mình nghĩ cũng có thể dịch là `phương trình pháp tuyến` hay `công thức pháp tuyến`.
Mọi người thấy sao nhỉ? @mlbvn/handson-ml2

**Comment by tiepvupsu** [comment, marked as answer]
Mình vote cho `* pháp tuyến nhé`. Từ `equation` trong tiếng anh có cả hai nghĩa `phương trình` và `công thức`. Mình dịch theo từng trường hợp.
